maestrano-connector-rails.gemspec

```diff
@@ -29,6 +29,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency('attr_encrypted', '~> 1.4.0')
   s.add_runtime_dependency('autoprefixer-rails')
   s.add_runtime_dependency('bootstrap-sass')
+  s.add_runtime_dependency 'health_check', '~> 2.4'
```

Comment:
same issue is in mno-enterprise 4.0 & 5.0, you want me to raise a pull request to fix there as well?

Reply:
No that's fine, it should install 2.8 on new projects and it was manually locked on older projects.
It's just because we adding it here we might as well put the latest version